src/app/components/user/Me.js

```diff
@@ -19,7 +19,7 @@ const Me = (props) => {
       <Relay.RootContainer
         Component={MeContainer}
         route={route}
-        renderLoading={() => <MediasLoading theme="grey" variant="page" size="large" />}
+        renderLoading={() => <MediasLoading theme="white" variant="page" size="large" />}
```

Comment:
This isn't part of this PR fix really, it was just already saved in my branch and is simply a style update, reviewers can ignore